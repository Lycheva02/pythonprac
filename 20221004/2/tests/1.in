(5,6),(5,7),(4,8),(11,1) 
