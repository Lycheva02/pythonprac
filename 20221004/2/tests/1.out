((5, 7), (4, 8), (11, 1))
